Esri/arcgis-maps-sdk-swift-samples
# [New] Augment reality to navigate route

## Description

This PR implements `Augment reality to navigate route` in `Augmented Reality` category.
URL to README: https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/destiny/Navigate-in-AR-sample/Shared/Samples/Augment%20reality%20to%20navigate%20route#readme

## Linked Issue(s)

- `swift/issues/4969`

## How To Test
- Select route in route planner view and tap camera to show AR view
- Route polyline should render over camera feed in AR.
- Start navigation to see route directions.

## Screenshots
![augment-reality-to-navigate-route](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/1243c43a-9c5b-469e-83aa-9bb205cdb73b)
